migration/src/m0000350_remove_old_assertion_tables.rs

```diff
@@ -0,0 +1,236 @@
+use crate::m0000040_create_vulnerability::Vulnerability;
+use crate::m0000060_create_advisory::Advisory;
+use crate::m0000100_create_package::Package;
+use crate::m0000120_create_package_version::PackageVersion;
+use crate::m0000140_create_package_version_range::PackageVersionRange;
+use crate::m0000150_create_affected_package_version_range::AffectedPackageVersionRange;
+use crate::m0000160_create_fixed_package_version::FixedPackageVersion;
+use crate::m0000170_create_not_affected_package_version::NotAffectedPackageVersion;
+use crate::Now;
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_table(
+                Table::drop()
+                    .table(AffectedPackageVersionRange::Table)
+                    .if_exists()
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
+            .drop_table(
+                Table::drop()
+                    .table(NotAffectedPackageVersion::Table)
+                    .if_exists()
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
+            .drop_table(
+                Table::drop()
+                    .table(FixedPackageVersion::Table)
+                    .if_exists()
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
+            .drop_table(
+                Table::drop()
+                    .table(PackageVersionRange::Table)
+                    .if_exists()
+                    .to_owned(),
+            )
+            .await?;
+
+        Ok(())
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
```

Comment:
I for one appreciate this level of effort, but when an `up()` consists mostly of `.drop_table`'s and its `down()` a bunch of `.create_table`'s -- especially for a feature that doesn't actually work -- I might be inclined to forget the `down()` as long as the old migrations drop their tables conditionally, i.e. if they exist.

Maybe. I'm not sure. Just looks weird. Forget I mentioned it.